tests/system_command/system_command.slt

```diff
@@ -1,15 +1,17 @@
+control substitution on
+
 system ok
-echo 114514 > __TEST_DIR__/test.txt
+echo 114514 > ${__TEST_DIR__}/test.txt
```

Comment:
What about removing the prefix and suffix underline? It looks more like an environment variable this way.

```suggestion
echo 114514 > $TEST_DIR/test.txt
```

Reply:
I'm afraid it can be confused with a normal environment variable. 🥵

Reply:
reasonable🥵